README.md

```diff
@@ -100,14 +100,19 @@ To generate a tarball archive that includes the agent binary and a configuration
 
 ### Windows
 
-To generate an MSI that will install the agent as a Windows service using a configuration file compatible with Google Cloud Monitoring:
+To build the source locally:
 
-1. Ensure that [Chocolatey](https://docs.chocolatey.org/en-us/) is [installed](https://docs.chocolatey.org/en-us/choco/setup)
-2. Run `.build\msi\make.ps1 Install-Tools` to install the open source [WIX Toolset](https://wixtoolset.org).
-3. Run `.build\msi\make.ps1 New-MSI`.
-4. The MSI file will be generated in the `dist` folder.
```

Comment:
Actually, it should be fairly straightforward to make the original MSI instructions work.
(a) they expect the binary to have already been built (in `bin/google-cloud-metrics-agent_windows_amd64.exe`);
(b) the `setx` command in `Install-Tools` in `make.ps1` is indeed broken (it needs to use `$env:PATH` instead of `%PATH%`;
(c) you have to start a new Administrator PowerShell between steps 2 and 3 to pick up the changes in the environment from step 2.
I'll let you decide if this is worth pulling into this PR.